### PR TITLE
Include all data from charts to csv file

### PR DESCRIPTION
**Pull request by echaidemenos**
The purpose of this PR is to close https://github.com/aqualinkorg/aqualink-app/issues/704

Changes the position of `Download CSV` button in site view. Adds a dialog with checkboxes from all non zero values for the user to download. It downloads only data between the selected dates on the charts.

Screenshot from 2022-05-05 17-52-43

Screenshot from 2022-05-19 17-39-45

Example output
```
﻿timestamp,satelliteTemperature_noaa,windSpeed_spotter,significantWaveHeight_spotter,topTemperature_spotter,bottomTemperature_spotter,sstAnomaly_noaa
2022-04-18T21:00:00.000Z,16.11,,,,,
2022-04-27T21:00:00.000Z,,14.4,0.82,,,
2022-04-27T22:00:00.000Z,,12.96,0.78,20.906666666666666,20.86,
2022-04-27T23:00:00.000Z,,12.96,0.84,20.860000000000003,20.86,
2022-04-28T00:00:00.000Z,,14.4,0.68,20.80666666666667,20.880000000000003,
2022-04-28T01:00:00.000Z,,12.96,0.71,20.773333333333333,20.873333333333335,
2022-04-28T02:00:00.000Z,,12.96,0.65,20.766666666666666,20.86,
2022-04-28T03:00:00.000Z,,12.96,0.74,20.78,20.880000000000003,
2022-04-28T04:00:00.000Z,,14.4,0.7,20.78,20.886666666666667,
2022-04-28T05:00:00.000Z,,15.840000000000002,0.72,20.78,20.880000000000003,
2022-04-28T06:00:00.000Z,,18.720000000000002,0.79,20.766666666666666,20.873333333333335,
2022-04-28T07:00:00.000Z,,20.16,0.74,20.753333333333334,20.846666666666668,
2022-04-28T08:00:00.000Z,,25.92,1.04,20.726666666666667,20.840000000000003,
2022-04-28T09:00:00.000Z,,20.16,0.82,20.700000000000003,20.826666666666668,
2022-04-28T10:00:00.000Z,,5.760000000000001,0.7,20.700000000000003,20.82,
2022-04-28T11:00:00.000Z,,4.32,0.65,20.700000000000003,20.800000000000004,
2022-04-28T12:00:00.000Z,,4.32,0.71,20.700000000000003,20.800000000000004,
2022-04-28T13:00:00.000Z,,4.32,0.72,20.706666666666667,20.82,
2022-04-28T14:00:00.000Z,,2.8800000000000003,0.69,20.720000000000002,20.82,
2022-04-28T15:00:00.000Z,,2.8800000000000003,0.71,20.78,20.840000000000003,
2022-04-28T16:00:00.000Z,,2.8800000000000003,0.64,20.833333333333336,20.853333333333335,
2022-04-28T17:00:00.000Z,,2.8800000000000003,0.61,20.846666666666668,20.880000000000003,
2022-04-28T18:00:00.000Z,,2.8800000000000003,0.71,20.873333333333335,20.913333333333334,
2022-04-28T19:00:00.000Z,,2.8800000000000003,0.67,20.9,20.926666666666666,
2022-04-28T20:00:00.000Z,,4.32,0.68,20.9,20.953333333333333,
2022-04-28T21:00:00.000Z,,2.8800000000000003,0.66,20.913333333333334,20.973333333333336,
2022-04-28T22:00:00.000Z,,10.08,0.55,20.9,20.986666666666668,
2022-04-28T23:00:00.000Z,,7.2,0.55,20.860000000000003,20.980000000000004,
2022-04-29T00:00:00.000Z,,8.64,0.55,20.840000000000003,20.953333333333333,
2022-04-29T01:00:00.000Z,,8.64,0.55,20.826666666666668,20.926666666666666,
2022-04-29T02:00:00.000Z,,11.520000000000001,0.5,20.793333333333337,20.9,
2022-04-29T03:00:00.000Z,,11.520000000000001,0.51,20.78,20.893333333333334,
2022-04-29T04:00:00.000Z,,14.4,0.52,20.78,20.880000000000003,
2022-04-29T05:00:00.000Z,,12.96,0.51,20.766666666666666,20.866666666666667,
2022-04-29T06:00:00.000Z,,14.4,0.56,20.759999999999998,20.880000000000003,
```

### CODE REVIEW

**Comment by echaidemenos** [comment]
I am not sure if we should have some static names for the columns or dynamical create the names based on `chartTitle` and info from `datasets`

**Comment by ericboucher** [comment]
@echaidemenos in the python SDK with standardize with this format: `"{metric_name}_{sonde_type_name}"` so I think we should simply do the same here.

**Comment by ericboucher** [comment]
@echaidemenos let's just download all the data by default for now. No need to ask them to select.

**Comment by echaidemenos** [comment]
> @echaidemenos in the python SDK with standardize with this format: `"{metric_name}_{sonde_type_name}"` so I think we should simply do the same here.

@ericboucher whats the `sonde_type_name` for monthly mean satellite temperature  ?

**Comment by ericboucher** [comment]
> @ericboucher whats the `sonde_type_name` for monthly mean satellite temperature ?

Let's use `noaa`

File: packages/website/src/common/Chart/MultipleSensorsCharts/index.tsx [hunk 1]
```diff
@@ -397,6 +391,50 @@ const MultipleSensorsCharts = ({
...
+    ...tempAnalysisDatasets.map((dataset) => ({
+      name: `${dataset.metric || "unknownMetric"}_${
+        dataset.source || "unknownSource"
+      }`,
+      values: dataset.data,
+    })),
+    ...Object.entries(spotterConfig).map(([key]) => {
+      const dataset = spotterMetricDataset(key as Metrics);
+      return {
+        name: `${key}_spotter`,
```
[the remark this answers is not in the document]
Comment (echaidemenos) [reply packages/website/src/common/Chart/MultipleSensorsCharts/index.tsx:411]: because `key` is `Metrics` and `type Metrics = CamelCase<MetricsKeys>;`

**Review by ericboucher — APPROVED** [review]

**Pull request merged**